### DISCUSSION TITLE
GPU/CUDA/OpenCV/TensorFlow?

### DISCUSSION THREAD

**Discussion by strarsis** [blakeblackshear/frigate#2214] [General]
Can Frigate use my NVIDIA GPU instead of the CPU for AI stuff? I read that one can connect a DeepStack server with it (DoubleTake addon), that would solve my question, too, as DeepStack GPU edition will use the GPU.
I don't have a Google Coral TPU, as cool as it is and as I would want to own one. So I am of course interested in offloading as much as possible into the GPU.

**Comment by blakeblackshear** [comment]
Frigate can't use an NVIDIA GPU. It will require significant changes.
